boards/nordic/nrf52dk_base/Cargo.toml

```diff
@@ -4,6 +4,10 @@ version = "0.1.0"
 authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
 edition = "2018"
 
+[features]
```

Comment:
Is there a way to document what these features mean? I.e. does TOML support comments? Would be good to not force people to go back and forth with the data sheet.

Reply:
Actually, the `ap_protect` is an attempt (but still WIP) to build a "production" firmware that would disable JTAG and set other read-back protections on. This is why this should be a feature. But we're not using it yet so we can also remove it from this PR without any issue if you prefer.

The `nfc_as_gpio` should probably be moved to a parameter set by the board instead of a feature. On Nordic, boards, this can be passed to the `setup_board()` function, like the voltage regulator setting. Its role is to tell whether the NFC antennas pins are used as regular GPIOs or have an antenna attached to them (in such case, some internal high voltage protection should be switched on which is the default case).

Reply:
Ah, great! Looking forward to that!

Reply:
Marking this conversation as resolved as the feature was replaced by a `setup_board` parameter.